client/jetpack-cloud/sections/agency-dashboard/sites-overview/test/test-utils/constants.ts

```diff
@@ -50,10 +50,17 @@ const site: Site = {
 		mobile: 75,
 		desktop: 85,
 	},
-	php_version_num: 7.4,
-	is_connected: true,
+	php_version_num: '7.4',
```

Comment:
The `php_version_num` field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.